examples/clipboard.py

```diff
@@ -0,0 +1,24 @@
+"""Copy canvas or whole viewer to clipboard."""
+from skimage import data
+
+import napari
+from napari._qt.widgets.qt_viewer_buttons import QtViewerPushButton
+
+
+# create the viewer with an image
+viewer = napari.view_image(data.cells3d())
+
+layer_buttons = viewer.window.qt_viewer.layerButtons
+
+# Add button to copy image of the canvas to clipboard.
```

Comment:
I'm not so sure about this example - i find the buttons with the warning symbol a little confusing to connect to the copy to clip board functionality. Maybe we just do without it as the functionality is in a dropdown menu?

Reply:
Thanks for the feedback - perhaps it wasn't the best example. I've modified it to add include buttons with text in a side dock instead.